src/gui/MainWindow.cc

```diff
@@ -1777,14 +1778,11 @@ bool MainWindow::eventFilter(QObject *obj, QEvent *event)
 
 void MainWindow::setRenderVariables(ContextHandle<BuiltinContext>& context)
 {
-  context->set_variable("$preview", Value(this->is_preview));
-  context->set_variable("$t", Value(this->animateWidget->getAnim_tval()));
-  auto camVpt = qglview->cam.getVpt();
-  context->set_variable("$vpt", Value(VectorType(context->session(), camVpt.x(), camVpt.y(), camVpt.z())));
-  auto camVpr = qglview->cam.getVpr();
-  context->set_variable("$vpr", Value(VectorType(context->session(), camVpr.x(), camVpr.y(), camVpr.z())));
-  context->set_variable("$vpd", Value(qglview->cam.zoomValue()));
-  context->set_variable("$vpf", Value(qglview->cam.fovValue()));
+  RenderVariables r;
+  r.preview = this->is_preview;
+  r.time = this->animateWidget->getAnim_tval();
+  r.camera = qglview->cam;
```

Comment:
RenderVariables r = {
 .preview = this->is_preview,
 .time = this->animateWidget->getAnim_tval(),
 .camera = qglview->cam,
};

Reply:
Sure.

Reply:
And the equivalent in openscad.cc, I think.

Reply:
Done.